tools/travis-setup.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+set -e
+
+# Skip this setup for jobs that don't run exunit
+test "${PRESET}" == "exunit" || exit 0
+
+## Generate fake certs
+mix certs.dev
+
+## Start mocks
+docker run -d -p 2197:2197 mobify/apns-http2-mock-server
+docker run -d -p 443:4000 4001:4001 kmakiela/fcm-mock-server
```

Comment:
```suggestion
docker run -d -p 4001:4001 kmakiela/fcm-mock-server
```

... since we don't use the HTTPS one.

Reply:
We use it, i.e here:
https://github.com/esl/MongoosePush/blob/da06ae7a76a762d4b21e67b9558dab8e4943e3b2/test/mongoose_push_test.exs#L371
and
https://github.com/esl/MongoosePush/blob/da06ae7a76a762d4b21e67b9558dab8e4943e3b2/.circleci/config.yml#L35
Should we change that?

Reply:
Didn't remember that, but yeah, let's use HTTP only reduce complexity.